metrics/exp/doc.go

```diff
@@ -14,6 +14,6 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Hook go-metrics into expvar
-// on any /debug/metrics request, load all vars from the registry into expvar, and execute regular expvar handler
+// package exp hooks go-metrics into expvar on any metrics request,
```

Comment:
Shouldn't the first letter be capitalized?

Reply:
Right! I will update this.